ui/less/overflow_menu.less

```diff
@@ -55,6 +55,11 @@
       background: rgb(224, 224, 224);
     }
 
+    /* When buttons are not aria-disabled, those will indicate as pointers */
+    &:not([aria-disabled=true]), &:not([aria-disabled=true]) label {
```

Comment:
I think [aria-disabled](https://a11y-101.com/development/aria-disabled) is more for accessibility features.

Reply:
Indeed! that's why we should omit them and make them WCAG-compliant. 

for example, if someone configured a button(for assistive device) with aria-disabled=true, we should not present them as action attached element (show cursor pointer).

In any case, If that is not completely relevant here. we can just apply the cursor pointer to the button element and the label. What do you think?

Reply:
We never disable any of the UI elements, so I don't think the condition is needed.

Reply:
Also, I'd suggest adding a comment about these items being clickable so we should use a pointer cursor.

Reply:
Thanks for the comment, I have removed the condition and added button and label to cursor pointer.